src/scenes/Users/List/UserList.tsx

```diff
@@ -0,0 +1,56 @@
+import { Grid, makeStyles, Typography } from '@material-ui/core';
+import React, { FC } from 'react';
+import { User } from '../../../api';
+import { ListContainer } from '../../../components/ListContainer';
+import { SortButtonDialog, useSort } from '../../../components/Sort';
+import { UserListItemCard } from '../../../components/UserListItemCard';
+import { listOrPlaceholders } from '../../../util';
+import { useUsersQuery } from './users.generated';
+import { UserSortOptions } from './UserSortOptions';
+
+const useStyles = makeStyles(({ spacing }) => ({
+  options: {
+    margin: spacing(3, 0),
+  },
+  projectItem: {
+    marginBottom: spacing(2),
+  },
+}));
+
+export const UserList: FC = () => {
+  const sort = useSort<User>();
+
+  const { data } = useUsersQuery({
+    variables: {
+      input: {
+        ...sort.value,
+      },
+    },
+  });
+  const classes = useStyles();
+
+  return (
+    <ListContainer>
+      <Typography variant="h2" paragraph>
+        People
+      </Typography>
+      <Grid container spacing={1} className={classes.options}>
+        <Grid item>
+          <SortButtonDialog {...sort}>
+            <UserSortOptions />
+          </SortButtonDialog>
+        </Grid>
+      </Grid>
+      <Typography variant="h3" paragraph>
+        {data?.users.total} People
```

Comment:
Hmm I know the other lists don't have this but it would be probably good to have a skeleton here while loading. Maybe 2-3ch width.

Reply:
👍